Aleksandr Zorin/css/style.css

```diff
@@ -0,0 +1,120 @@
+@charset "utf-8";
+/* CSS Document */
+
+body {
+	background-image: url(../img/background.jpg);
+	background-position: center center;
+	background-repeat: no-repeat;
+	background-attachment: fixed;
+	background-size: cover;
+	font-family: 'Dosis', sans-serif;
+}
+
+hr {
+	color: #8E4140;
+	background-color: #8E4140;
+	height: 1px;
+}
+
+p {
+	margin: 0;
+}
+
+.weatherWidget {
+	margin-top: 100px;
+	display: flex;
+	justify-content: center;
+	flex-wrap: wrap;
+}
+
+.location {
+	background-image: url(../img/city.jpg);
+	background-size: cover;
+	width: 500px;
+	height: 750px;
+	border-radius: 10px 0 0 10px;
+	box-shadow: 0px 0px 100px 1px black;
+}
+
+.information {
+	width: 400px;
+	height: 750px;
+	padding-left: 50px;
+	padding-right: 50px;
+	background-color: #FFFFFF;
+	border-radius: 0px 10px 10px 0px;
+	box-shadow: 0px 0px 50px 1px black;
+	display: flex;
+	flex-direction: column;
+}
+
+.temp {
+	height: 25%;
+	padding-top: 80px;
+	font-size: 100pt;
+	font-weight: 200;
+	line-height: 0.5;
+	color: #8E4140;
+	text-align: center;
+}
+
+.graph {
+	height: 25%;
+	margin-bottom: 20px;
+}
+
+.days {
+	display: flex;
+	justify-content: space-around
+}
+
+.weatherType {
+	height: 20%;
+	margin-top: -85px;
+	background: rgba(46,28,37,0.89);
+	font-size: 60pt;
+	line-height: 1.5;
+	font-weight: 200;
+	text-align: center;
+	color: white;
+}
+
+.today {
+	background: rgba(119,25,33,0.89);
+	height: 20%;
+	font-size: 20pt;
+	font-weight: 300;
+	line-height: 5;
+	text-align: center;
+	color: white;
+}
+
+.weatherTypeLogo {
+	height: 60%;
```

Comment:
@Zorin-Alex Please try to avoid of mark-up with percentage for .location block. You can use padding-bottom to set needed offset below the image